.ci/pipeline/ci.yml

```diff
@@ -382,10 +382,10 @@ jobs:
       .ci/scripts/describe_system.sh
     displayName: 'System info'
   - script: |
-      .ci/scripts/build.sh --compiler clang --target daal --optimizations "sse2 sse42 avx2 avx512" --conda-env ci-env
```

Comment:
I think it would be cleaner to have this in a separate commit. Why was it disabled?

Reply:
@ahuber21 
Previously, subset of ISAs didn't work correctly for building on macOS and all of them were selected in CI. Now, it's changed to SSE2+AVX2